components/input/slider/rangeslider/RangeSlider.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useRef } from 'react';
+import styles from './RangeSlider.module.css';
+
+function RangeSlider() {
+	const [rangeValues, setRangeValues] = useState({
```

Comment:
Should use two different state values. On changing min value, max value component will re-render unnecessarily and vice versa.